minecraft_servers/kettlemc/manifest.json

```diff
@@ -0,0 +1,79 @@
+{
+  "server_name": "kettlemc",
+  "nice_name": "KettleMC.net",
+  "direct_ip": "KettleMC.net",
+  "server_wildcards": [
+    "%.kettlemc.de",
+    "%.kettlemc.net",
+    "%.kettlemc.eu"
+  ],
+  "supported_languages": [
+    "de", 
+	"en"
+  ],
+  "social": {
+    "web": "https://www.kettlemc.net",
```

Comment:
Website not available 
<img width="838" alt="Bildschirm­foto 2023-01-27 um 16 50 12" src="https://user-images.githubusercontent.com/45363287/215128856-843ef811-02de-4812-ac9a-8f3e4a67c34d.png">

Reply:
Fixed